### PR TITLE
In emberAfEndpointConfigure() change endpoint ID to a uint16_t

### PR DESCRIPTION
**Pull request by selissia**
#### Problem
In emberAfEndpointConfigure() a uint8_t is used for the endpoint ID. According to the spec (7.18.2. Derived Data Types) a uint16_t must be used.

Separate minor issue: wrong type is being passed to OnOffServer::setOnOffValue()

#### Change overview
In emberAfEndpointConfigure() use uint16_t for the endpoint ID.

Change the type of the second parameter of OnOffServer::setOnOffValue() to chip::CommandId

#### Testing
Tested with Silabs Unify Matter bridge on Ubuntu

### CODE REVIEW

File: src/app/clusters/level-control/level-control.cpp [hunk 1]
```diff
@@ -311,7 +311,8 @@ static void setOnOffValue(EndpointId endpoint, bool onOff)
     if (emberAfContainsServer(endpoint, OnOff::Id))
     {
         emberAfLevelControlClusterPrintln("Setting on/off to %p due to level change", onOff ? "ON" : "OFF");
-        OnOffServer::Instance().setOnOffValue(endpoint, (onOff ? OnOff::Commands::On::Id : OnOff::Commands::Off::Id), true);
+        OnOffServer::Instance().setOnOffValue(
+            endpoint, static_cast<uint8_t>(onOff ? OnOff::Commands::On::Id : OnOff::Commands::Off::Id), true);
```
[the remark this answers is not in the document]
Comment (selissia) [reply src/app/clusters/level-control/level-control.cpp]: Done

**Review by jmartinez-silabs — APPROVED** [review]

**Review by jmartinez-silabs — APPROVED** [review]

File: src/app/util/attribute-storage.cpp [hunk 2]
```diff
@@ -129,7 +129,7 @@ static uint16_t findClusterEndpointIndex(EndpointId endpoint, ClusterId clusterI
 // Initial configuration
 void emberAfEndpointConfigure(void)
 {
-    uint8_t ep;
+    uint16_t ep;
```
Comment (mrjerryjohns) [inline src/app/util/attribute-storage.cpp:132]: Can you update `kTestEndpointId` and `kTestEndpointId3` in `TestReadChunking.cpp` to occupy values in the 16-bit space? That should give us unit-test coverage.

Please ensure that the test fails before this change.

Comment (mrjerryjohns) [reply src/app/util/attribute-storage.cpp:132]: Nvm. There is no code-path with dynamic endpoints that ends up affecting this logic here in `emberAfEndpointConfigure`.

Comment (mrjerryjohns) [reply src/app/util/attribute-storage.cpp:132]: Short of actually modifying the example apps to have endpoints in this space, there's no way to test this in unit-test :(

**Review by mrjerryjohns — APPROVED** [review]

**Pull request merged**